bundle/camunda-saas-bundle/Dockerfile

```diff
@@ -1,6 +1,12 @@
 FROM eclipse-temurin:17.0.8_7-jre
 
-RUN mkdir /opt/app
+# The /opt/app is used for the Connectors runtime, and out-of-the-box connectors
+# Use the /opt/additionalLibs to mount your own connectors, secret providers, or include other jars into the classpath
+RUN mkdir /opt/app && mkdir /opt/additionalLibs
```

Comment:
Can we use something simpler like `/opt/custom`?

Reply:
Just want to avoid any issues with snakeCaseUserProblems

Reply:
@sbuettner how about `/opt/extras/`?